src/services/generalServices.js

```diff
@@ -220,137 +337,17 @@ const exportToExcel = async (initialState, updatedState, prevSeasonYear, currSea
-                top: {style:'thin'},
-                left: {style:'thin'},
-                bottom: {style:'thin'},
-                right: {style:'thin'}
-            };
-
-            sheet.mergeCells('A4:L4')
-            sheet.getCell('A4').value = prevSeasonYear[0] + ' ' + prevSeasonYear[1] + ' to ' + currSeasonYear[0] + ' '  + currSeasonYear[1];
-            sheet.getCell('A4').font = {name: 'Calibri', size: 14, bold: true};
-            sheet.getCell('A4').alignment = { vertical: 'middle', horizontal: 'center' };
-            sheet.getCell('A4').fill = {
-                type: 'gradient',
-                gradient: 'angle',
-                degree: 0,
-                stops: [
-                    {position:0, color:{argb:'96C8FB'}},
-                    {position:0.5, color:{argb:'96C8FB'}},
-                    {position:1, color:{argb:'96C8FB'}}
-                ]
-            };
-            sheet.getCell('A4').border = {
-                top: {style:'thin'},
-                left: {style:'thin'},
-                bottom: {style:'thin'},
-                right: {style:'thin'}
-            };
-
-            sheet.mergeCells('A6:D6');
-            sheet.getCell('A6').value = 'General State Info';
-            sheet.getCell('A6').font = {name: 'Calibri', size: 14, bold: true};
-            sheet.getCell('A6').alignment = { vertical: 'middle', horizontal: 'center' };
-            sheet.getCell('A6').fill = {
-                type: 'gradient',
-                gradient: 'angle',
-                degree: 0,
-                stops: [
-                    {position:0, color:{argb:'C85C5C'}},
-                    {position:0.5, color:{argb:'F9975D'}},
-                    {position:1, color:{argb:'C85C5C'}}
-                ]
-            };
-            sheet.getCell('A6').border = {
-                top: {style:'thin'},
-                left: {style:'thin'},
-                bottom: {style:'thin'},
-                right: {style:'thin'}
-            };
-
-            sheet.mergeCells('G6:L6');
-            sheet.getCell('G6').value = 'Region Info';
-            sheet.getCell('G6').font = {name: 'Calibri', size: 14, bold: true};
-            sheet.getCell('G6').alignment = { vertical: 'middle', horizontal: 'center' };
-            sheet.getCell('G6').fill = {
-                type: 'gradient',
-                gradient: 'angle',
-                degree: 0,
-                stops: [
-                    {position:0, color:{argb:'C85C5C'}},
-                    {position:0.5, color:{argb:'F9975D'}},
-                    {position:1, color:{argb:'C85C5C'}}
-                ]
-            };
-            sheet.getCell('G6').border = {
-                top: {style:'thin'},
-                left: {style:'thin'},
-                bottom: {style:'thin'},
-                right: {style:'thin'}
-            };
+            await cellFormating(sheet, 'A1', 'L3', initialState[i].stateName, ['C85C5C','F9975D','C85C5C'])

```

Comment:
What are these three magic strings? `C85C5C` and all

Reply:
those are color hexcodes @Phobez

Reply:
Please extract them into more meaningful names, like `RED`